pkg/crypto/key.go

```diff
@@ -138,3 +138,56 @@ func (k *AESKey) Flush() error {
 
 	return os.WriteFile(getKeyPath(), io, 0644)
 }
+
+func (k *AESKey) ID() string {
+	sha := sha256.Sum256([]byte(k.Key))
```

Comment:
do we have any utils code for sha'ing currently?  I feel like it's around somewhere

Reply:
no, we don't have